symbolic-demangle/src/lib.rs

```diff
@@ -176,6 +188,12 @@ fn try_demangle_cpp(ident: &str, opts: DemangleOptions) -> Option<String> {
     {
         use cpp_demangle::{DemangleOptions as CppOptions, Symbol as CppSymbol};
 
+        // Strip away the hash suffixes described in
+        // https://github.com/gimli-rs/cpp_demangle/pull/210.
+        // Why the empty string needs a type annotation I have no earthly idea.
+        let empty: &[u8] = b"";
+        let ident = HASH_SUFFIX.replace(ident.as_bytes(), empty);
```

Comment:
Would be nice to avoid `regex` here, as that is a huge dependency. I think you should be able to just split off the suffix and check its validity.